### PR TITLE
Editorial: motivate API-as-a-Product rule

### PR DESCRIPTION
**Pull request by tfrauenstein**

### CODE REVIEW

File: chapters/design-principles.adoc [hunk 1]
```diff
@@ -22,9 +22,11 @@ make them irresistible for client engineers
 * Actively improve and maintain API consistency over the long term
 * Make use of customer feedback and provide service level support
 
-RESTful API as a Product makes the difference between enterprise
-integration business and agile, innovative product service business
-built on a platform of APIs.
+Embracing API as a Product facilitates a service ecosystem which can
```
Comment (whiskeysierra) [inline chapters/design-principles.adoc]: > which can be evolved more easily

> which can be easier to evolve

> which is easier to evolve

File: chapters/design-principles.adoc [hunk 2]
```diff
@@ -22,9 +22,11 @@ make them irresistible for client engineers
 * Actively improve and maintain API consistency over the long term
 * Make use of customer feedback and provide service level support
 
-RESTful API as a Product makes the difference between enterprise
-integration business and agile, innovative product service business
-built on a platform of APIs.
+Embracing API as a Product facilitates a service ecosystem which can 
+be easier evolved, and used to experiment quickly with new business 
+ideas by recombining core capabilities. It makes the difference between
```
Comment (whiskeysierra) [inline chapters/design-principles.adoc]: The last sentence was basically already there before, but I believe it needs some more attention.

> [ordinary] system integration business

> [innovative] product service business

The adjectives tell me how I should feel about them, but those terms don't really speak to me. What are we comparing here?

File: chapters/design-principles.adoc (hunk 1 above)
Comment (tfrauenstein) [reply chapters/design-principles.adoc]: done, thx.

File: chapters/design-principles.adoc (hunk 2 above)
Comment (tfrauenstein) [reply chapters/design-principles.adoc]: @whiskeysierra I'm comparing a situation where APIs are provided as "appendix" to existing systems to facilitate system integration (optimised for local server-side realization) in contrast to APIs as a product, optimised for easy usage, extensibility and open business adoption. 
You don't like my statement? Any concrete proposal how to improve?

Comment (whiskeysierra) [reply chapters/design-principles.adoc]: > I'm comparing a situation where APIs are provided as "appendix" to existing systems to facilitate system integration (optimised for local server-side realization) in contrast to APIs as a product, optimised for easy usage, extensibility and open business adoption.

This one is great. Can we rephrase the current sentence to make use of this one?

Comment (tfrauenstein) [reply chapters/design-principles.adoc]: 👍

**Comment by tlossen** [comment]
👍

**Comment by maxim-tschumak** [comment]
:+1:

**Pull request merged**